vulkan_generator/main.py

```diff
@@ -17,7 +17,7 @@
 from pathlib import Path
 import sys
 
-import vulkan_generator
+from gapid.vulkan_generator import vulkan_generator
```

Comment:
I was telling Melih that vulkan_generator shouldn't be in the base namespace and we should have some enclosing namespace for all the new project Sunrise code but I'm not sure we should use the name gapid. Since the tool is now called AGI maybe we could call it agi or agir (to copy the gapir idiom but with the new product name?)

Reply:
moved it to generator.py, so that it can be properly imported.